arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/StackedSet.java

```diff
@@ -56,9 +60,17 @@ public void pop() {
   }
 
   public boolean add(E e) {
-    if (!operationsCommitedToTheConflation().contains(e)) {
-      return operationsInTransaction().add(e);
+    return operationsInTransaction().add(e);
+  }
+
+  void deleteDuplicate() {
```

Comment:
The method name should be `deleteDuplicates`

Reply:
done